src/api_server/swagger/firecracker.yaml

```diff
@@ -774,7 +774,7 @@ definitions:
           snapshot is created.
       version:
         type: string
-        description: The snapshot format version.
+        description: The microVM version. It is optional and it defaults to the current version.
```

Comment:
Can we add a short explanation here what `microVM version` means (the version at which we want to save the microVM snapshot or something like this)?

Reply:
Done.